packages/stark-build/config/build-utils.js

```diff
@@ -144,7 +160,7 @@ function getCopyWebpackPluginConfig(assets) {
 		// specify a configuration flag.
 		// Also prevent writing outside the project path. That is not overridable.
 		// For info: Comparing to implementation of Angular, "buildOptions.outputPath" has been replaced by "appConfig.outDir"
-		const absoluteOutputPath = path.resolve(projectRoot, appConfig.outDir);
+		const absoluteOutputPath = path.resolve(projectRoot, appConfig["architect"]["build"]["options"]["outputPath"]);
```

Comment:
Again, I think we should just use the dot syntax whenever possible...

Reply:
done